service/controller/internal/unittest/default_controller_context.go

```diff
@@ -113,6 +113,7 @@ func DefaultContext() context.Context {
 					ARN: "peer-role-arn",
 				},
 				VPC: controllercontext.ContextStatusControlPlaneVPC{
+					ID:   "vpc-testid",
 					CIDR: "10.1.0.0/16",
```

Comment:
Super nitty, please order alphabetically as you actually did in the struct definition. 🙈